frontend/src/pages/Page.tsx

```diff
@@ -44,7 +45,10 @@ const Page = ({ children }: Props) => {
       {!isAuthenticated ? (
         <Alert severity="error">You are not part of this project.</Alert>
       ) : (
-        children
+        <Stack direction="row" height="100%" width="100vw" overflow="auto">
```

Comment:
can't we do with 100% here? i don't like vw that much

Reply:
Actually looks like we don't need the width